plat/nvidia/tegra/platform.mk

```diff
@@ -1,9 +1,22 @@
 #
-# Copyright (c) 2015-2017, ARM Limited and Contributors. All rights reserved.
+# Copyright (c) 2015-2018, ARM Limited and Contributors. All rights reserved.
 #
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
+# override with stdlib files used by the Tegra platform
+STDLIB_SRCS	:=	$(addprefix lib/stdlib/,	\
```

Comment:
This can't be working, the folder is now called libc. ``tf_snprintf.c`` doesn't exist anymore.